cmd/restic/global.go

```diff
@@ -280,7 +280,7 @@ func readPassword(in io.Reader) (password string, err error) {
 	sc := bufio.NewScanner(in)
 	sc.Scan()
 
-	return sc.Text(), errors.Wrap(err, "Scan")
+	return sc.Text(), errors.WithStack(sc.Err())
```

Comment:
I'd prefer to use `fmt.Errorf("....: %w", sc.Err())` to eventually get rid of the legacy errors package. Although, we could defer to clean up to some point in the future.